FunctionalTableDataDemo/View Controllers/TableExampleController.swift

```diff
@@ -11,28 +11,25 @@ import FunctionalTableData
 
 class TableExampleController: UITableViewController {
 	private let functionalData = FunctionalTableData()
-	private var items: [String] = [] {
-		didSet {
-			render()
-		}
-	}
+	private var items: [String] = []
 	
 	override func viewDidLoad() {
 		super.viewDidLoad()
-		
+		tableView.isEditing = true
 		functionalData.tableView = tableView
 		title = "Table View"
 		navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(didSelectAdd))
 	}
 	
 	@objc private func didSelectAdd() {
-		items.append(NSDate().description)
+		items.append("\(items.count)")
+		render()
 	}
 	
 	private func render() {
 		let rows: [CellConfigType] = items.enumerated().map { index, item in
 			return LabelCell(
-				key: "id-\(index)",
+				key: "id-\(item)",
```

Comment:
why this change? index helps with uniquing, I don't suggest making that change.

Reply:
Will remove this change

The intention was to have a manual sorting simple sample code with moves detection. Keeping the index as an id makes keys vary for each object on each render when reordering. FunctionalTableData then detects an insertion and a deletion instead of a move.
Uniquing was possible as there is no way to delete a item from the array.

But not sure of the added value anymore, moreover with github.com/Shopify/FunctionalTableData/pull/147  removing this class really soon.

Reply:
With the item it will crash if we spam the button right? But I agree with not using index here. I don't think it matters. I want to delete these examples anyway

Reply:
Should not have crashed as `NSDate().description` was replaced by `items.count` but decided to revert the example changes